### ISSUE TITLE
Add constants for common defaults such as IID(Normal())

### ISSUE THREAD

**Issue by seabbs** [CDCgov/Rt-without-renewal#543]
The default version of this is so common, would it be worth having a constant like

```julia
const NormalIID = IID()
```

Which can be used in places like the `generate_latent` method for `HierarchicalNormal`?

_Originally posted by @SamuelBrand1 in https://github.com/CDCgov/Rt-without-renewal/pull/438#discussion_r1879847119_

**Comment by SamuelBrand1** [comment]
Yes. I'm abit agnostic, but I personally like const s for really commonly used objects. ymmv though.
